CMakeLists.txt

```diff
@@ -499,6 +499,26 @@ if (UNIX AND NOT APPLE)
 		    	COMPONENT Dependencies
 		    	)
 
+                    # Determine if we are building on Ubuntu 22.  If so, include xinerama
+		    #
+                    file(READ "/etc/issue" ETC_ISSUE)
+                    string(REGEX MATCH "Ubuntu" DIST ${ETC_ISSUE})
+                    if(DIST STREQUAL "Ubuntu")
+                        find_program(LSB_RELEASE_EXEC lsb_release)
+                        execute_process(COMMAND ${LSB_RELEASE_EXEC} -rs
+                            OUTPUT_VARIABLE LSB_RELEASE
+                            OUTPUT_STRIP_TRAILING_WHITESPACE
+                        )
+			if(${LSB_RELEASE} MATCHES "^22.")
+		            file (GLOB XCB_FILES ${THIRD_PARTY_LIB_DIR}/libxcb-xinerama.*)
+		            install (
+		        	FILES ${XCB_FILES}
+		        	DESTINATION ${INSTALL_LIB_DIR}
+		        	COMPONENT Dependencies
+		        	)
```

Comment:
Have you checked if the automated dependency generator script bundles the lib already?

Reply:
I suppose so, as my check was to see if the installer worked on Ubuntu 22 and that it included libxinerama.  I also tested builds on our other platforms in case this new logic broke their builds.